cypress/e2e/shared/tasks.test.ts

```diff
@@ -251,35 +246,24 @@ http.post(url: "https://foo.bar/baz", data: bytes(v: "body"))`
 
     it('can clone a task and edit it', () => {
       // clone a task
-      cy.getByTestID('task-card')
-        .trigger('mouseover')
-        .then(() => {
-          cy.get('.context-menu--container')
-            .eq(1)
-            .click()
-          cy.getByTestID('context-menu-item')
-            .contains('Clone')
-            .click()
-        })
+      cy.getByTestID('task-card').then(() => {
+        cy.getByTestID('context-menu-task').click()
+        cy.getByTestID('context-clone-task')
+          .click()
+          .type('{esc}')
```

Comment:
After clicking clone in the task list, the popover stays open. Typing ESC turns off the popover.